tripy/tripy/utils/ast.py

```diff
@@ -141,10 +137,6 @@ def index_into_expr(node: ast.expr, index: int) -> ast.expr:
             else:
                 arg_node = index_into_expr(node.slice, index)
 
-        # if the resulting arg node is a list or tuple literal, we might be able to narrow it down with a list index
-        if list_index is not None and isinstance(arg_node, (ast.List, ast.Tuple)) and list_index < len(arg_node.elts):
-            arg_node = arg_node.elts[list_index]
```

Comment:
What was the reason for getting rid of this? Did it cause problems somewhere?

Reply:
It was dead code

Reply:
Ah, I see, the code above handles it too.